entries/QUnit.module.xml

```diff
@@ -87,5 +124,43 @@ QUnit.test( "makes a car", function( assert ) {
 
 ]]></code>
 	</example>
+	<example>
+		<desc>Hooks stack on nested modules</desc>
+<code><![CDATA[
+QUnit.module( "grouped tests argument hooks", function( hooks ) {
+	var beforeEach = hooks.beforeEach;
```

Comment:
Also, why do you add these variables? Can you just call `hooks.beforeEach` below?